app/components/header.hbs

```diff
@@ -0,0 +1,69 @@
+<section class='header'>
+  <img
+    class='header__logo'
+    src='assets/images/rds-logo-2x.png'
+    alt='Real Dev Squad'
+  />
+
+  <div class='header__container'>
+    <span class='container__text-social'>
+      <h2 class='text__subtitle'>
+        <span class='text__welcome'>Welcome</span>
+        <span class='text__to'>to the</span>
+      </h2>
+      <h1 class='text__title'>Real Dev Squad</h1>
+
+      <div class='social'>
+        <a href={{this.LINKEDIN_URL}} target='_blank' class='social__handles'>
```

Comment:
Do you think we should be adding `noreferrer noopener` here https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types

Reply:
Yes, we should add that. Made the change